File: papi/lista2/exercicio4.py
print("Faremos a média de 4 notas")

while True:
    try:
        nota1 = int(input("Por favor informe a primeira nota (use o ponto como separador decimal): "))
        break
    except ValueError:
        print("Por favor informe um número válido.")

while True:
    try:
        nota2 = int(input("Por favor informe a segunda nota (use o ponto como separador decimal): "))
        break
    except ValueError:
        print("Por favor informe um número válido.")

while True:
    try:
        nota3 = int(input("Por favor informe a terceira nota (use o ponto como separador decimal): "))
        break
    except ValueError:
        print("Por favor informe um número válido.")

while True:
    try:
        nota4 = int(input("Por favor informe a quarta nota (use o ponto como separador decimal): "))
        break
    except ValueError:
        print("Por favor informe um número válido.")


print("A média das notas é", (nota1 + nota2 + nota3 + nota4) / 4)

File: papi/lista2/exercicio4.kt
package lista2

fun main(){
    var notas = mutableListOf<Double>()

    var count = 0

    while (count < 4) {
        try {
            print("Por favor informe uma nota: ")
            val input = readLine()
            notas.add(input!!.toDouble())
            count++           
        } catch (e: Exception) {
            println("Erro na conversão de string para número")
        }
    }

    val media = notas.sum() / notas.size
    print("A média das notas é $media")
}